posthog/api/test/test_action.py

```diff
@@ -188,6 +188,20 @@ def test_dau_filtering(self):
         self.assertEqual(response[0]['data'][4], 1)
         self.assertEqual(response[0]['data'][5], 2)
 
+    def test_dau_with_breakdown_filtering(self):
```

Comment:
will test that after adding another event with a previously existing distinct_id and property that the count is aggregated